support/scripts/patch-rpath.py

```
@@ -0,0 +1,116 @@
+from pathlib import Path
+
+import lief
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+lief.logging.set_level(lief.logging.LEVEL.ERROR)
+
+
+def patch_elf(binary: lief.ELF.Binary, rpaths: list[str]) -> None:
+	logger.info(f'Processing ELF binary')
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)